backend/api/Areas/Admin/Controllers/AdministrativeAreaController.cs

```diff
@@ -0,0 +1,180 @@
+using System.Threading.Tasks;
+using MapsterMapper;
+using Microsoft.AspNetCore.Mvc;
+using Pims.Api.Policies;
+using Pims.Dal.Security;
+using Pims.Dal.Services.Admin;
+using Swashbuckle.AspNetCore.Annotations;
+using Entity = Pims.Dal.Entities;
+using Model = Pims.Api.Areas.Admin.Models.AdministrativeArea;
+using EModel = Pims.Dal.Entities.Models;
+using Pims.Dal.Keycloak;
+
+namespace Pims.Api.Areas.Admin.Controllers
+{
+    /// <summary>
+    /// AdministrativeArea class, provides endpoints for managing administrative areas.
+    /// </summary>
+    [HasPermission(Permissions.AgencyAdmin)]
+    [ApiController]
+    [Area("admin")]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/[area]/administrativeAreas")]
+    [Route("[area]/administrativeAreas")]
+    public class AdministrativeAreaController : ControllerBase
+    {
+        #region Variables
+
+        private readonly IPimsAdminService _pimsAdminService;
+        private readonly IPimsKeycloakService _pimsKeycloakService;
+        private readonly IMapper _mapper;
+
+        #endregion
+
+        #region Constructors
+
+        /// <summary>
+        /// Creates a new instance of a AdministrativeAreaController class.
+        /// </summary>
+        /// <param name="pimsAdminService"></param>
+        /// <param name="keycloakService"></param>
+        /// <param name="mapper"></param>
+        public AdministrativeAreaController(IPimsAdminService pimsAdminService, IPimsKeycloakService keycloakService,
+            IMapper mapper)
+        {
+            _pimsAdminService = pimsAdminService;
+            _pimsKeycloakService = keycloakService;
+            _mapper = mapper;
+        }
+
+        #endregion
+
+        #region Endpoints
+
+        /// <summary>
+        /// GET - Returns a paged list of administrative areas from the datasource.
+        /// </summary>
+        /// <returns>Paged object with an area of administrative areas.</returns>
+        [HttpGet]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(Api.Models.PageModel<Model.AdministrativeAreaModel>), 200)]
```

Comment:
Says you're returning a Page of AdministrativeArea, but you are returning an array.

Reply:
Doesn't look like you fixed this.

Reply:
Still does not look like you've resolved this.
`[ProducesResponseType(typeof(Api.Models.PageModel<Model.AdministrativeAreaModel>), 200)]`
doesn't match your return
`return new JsonResult(_mapper.Map<Model.AdministrativeAreaModel[]>(adminAreas));`